setup.py

```diff
@@ -123,6 +123,49 @@ def get_cuda_path():
     return _cuda_path
 
 
+def get_sycl_path():
+    basekit = os.environ.get("CMPLR_ROOT")
+    if basekit:
+        return basekit
+    else:
+        # If basekit env is not set then try default path
+        return "/opt/intel/oneapi/compiler/latest/"
```

Comment:
Please mirror the corresponding CUDA version by checking whether paths exist (return `None` otherwise)

Reply:
updated